.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - "*.*.*"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.ivy2
+            ~/.sbt
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Build
+        run: sbt package
+      - uses: actions/upload-artifact@v2
+        with:
+          name: jars
+          path: 'play/target/*/*/*.jar'
+          retention-days: 1
+  publish:
+    if: github.event_name != 'pull_request'
+    needs: build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.ivy2
+            ~/.sbt
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Set SBT_OPTS
+        run: bash gen_sbt_opts.sh
+      - name: Publish to Sonatype
+        run: bash publish.sh
+        env:
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
```

Comment:
Have these been added yet? It'd be good to do a test publish before merging

Reply:
yes

Reply:
I am not certain how to do that.. Also, I would prefer to fix it on master if its broken rather than testing publish on a feature branch.